libs/core/src/lib/shellbar/shellbar-actions-mobile/shellbar-actions-mobile.component.ts

```diff
@@ -23,9 +25,17 @@ export class ShellbarActionsMobileComponent implements AfterContentChecked {
     @Input()
     collapsedItemMenuLabel: string;
 
+    /** @hidden */
+    @Input()
+    searchExists = false;
```

Comment:
This component is internal one, so I think it's fine to not document some stuff here

Reply:
Maybe we can mention this in comments? After hidden